### PR TITLE
ui tweaks

### PR DESCRIPTION
**Pull request by cmkohnen**
Make Dialogs consistent.
Also fix adding a style

### CODE REVIEW

File: src/export_figure.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ class ExportFigureWindow(Adw.Window):
     def __init__(self, application):
         super().__init__(
             application=application, transient_for=application.main_window,
-            file_formats=application.canvas.get_supported_filetypes_grouped(),
+            file_formats=application.main_window.toast_overlay.get_child(
+            ).get_supported_filetypes_grouped(),
```
Comment (Sjoerd1993) [inline src/export_figure.py:24]: Why this change? `get_supported_filetypes_grouped()` is a method from Matplotlibs canvas. It doesn't seem obvious why one would call it from the child of the toast overlay in the main window. (Which happens to be the same canvas)

**Review by Sjoerd1993 — COMMENTED** [review]
Just wondering about the comment I placed. Indeed does fix the broken add style.

File: src/export_figure.py (hunk 1 above)
Comment (cmkohnen) [reply src/export_figure.py:24]: We no longer store a reference to the actual canvas, as this is probably (one of) the only cases, where we would need to access it.
This is why it broke before (as I didn't catch it during search).

Comment (Sjoerd1993) [reply src/export_figure.py:24]: Ah gotcha, I didn't catch it either as I didn't think about the actual reference no longer being stored. That makes sense then.

**Review by Sjoerd1993 — APPROVED** [review]
